src/frontend/src/pages/GanttPage/GanttChartSection.tsx

```diff
@@ -62,47 +68,26 @@ const GanttChartSection = ({
       <Box sx={{ mt: '1rem', width: 'fit-content' }}>
         {projects.map((project) => {
           return (
-            <>
-              <Box display="flex" alignItems="center">
-                <GanttTaskBar
-                  key={project.id}
-                  days={days}
-                  event={project}
-                  isEditMode={isEditMode}
-                  createChange={handleCreateProjectChange}
-                  handleOnMouseOver={handleOnMouseOver}
-                  handleOnMouseLeave={handleOnMouseLeave}
-                  onWorkPackageToggle={() => toggleWorkPackages(project)}
-                  showWorkPackages={showWorkPackagesMap.get(project.id)}
-                  addWorkPackage={addWorkPackage}
-                  highlightedChange={
-                    highlightedChange && project.id === highlightedChange.eventId ? highlightedChange : undefined
-                  }
-                />
-              </Box>
-              <Collapse in={showWorkPackagesMap.get(project.id)}>
-                {project.workPackages.map((workPackage) => {
-                  return (
-                    <GanttTaskBar
-                      key={workPackage.id}
-                      days={days}
-                      event={workPackage}
-                      isEditMode={isEditMode}
-                      createChange={createChange}
-                      handleOnMouseOver={handleOnMouseOver}
-                      handleOnMouseLeave={handleOnMouseLeave}
-                      highlightedChange={
-                        highlightedChange &&
-                        (workPackage.id === highlightedChange.eventId ||
-                          highlightedChange?.eventId === workPackage.projectId)
-                          ? highlightedChange
-                          : undefined
-                      }
-                    />
-                  );
-                })}
-              </Collapse>
-            </>
+            <Box display="flex" alignItems="center">
+              <GanttTaskBar
+                key={project.id}
+                days={days}
+                task={transformProjectToGanttTask(project, teamName)}
```

Comment:
if we add team to ProjectPreview, should be able to save us the trouble of passing teamName down and to here as well

Reply:
its not possible :(